### DISCUSSION TITLE
Display html image tags in QLMarkdown?

### DISCUSSION THREAD

**Discussion by chopper7500** [sbarex/QLMarkdown#111] [Q&A] [answered]
First off thanks for keeping QLMarkdown alive - I use it regularly.
One issue I have is that while markdown images of the form `![alt text](image.png)` display fine in quicklook, I cannot seem to get the most basic html images to display - for example `<img src="image.png" alt="alt text">`. Does QLMarkdown just ignore html images?

Thanks,

Craig

**Comment by chopper7500** [comment]
Doh. Just found the "Inline HTML (unsafe)" option hidden under Advanced options. Is this just considered "unsafe" as there may be malicious html in a markdown file and QLMarkdown will happily execute it?

**Comment by sbarex** [comment, marked as answer]
The terminology is derived from that of `cmark` which is the rendering engine used. It is indicated as insecure both for the possibility of inserting malicious code (such as `<script>` or `<iframe>`) and for the possibility of interfering with the rendering of the remaining markdown code. If the `tag filter` option is checked in the advanced settings, then all potentially dangerous tags are disabled (such as `<script>`).

**Reply by chopper7500** [reply]
I see that these options are described in the README file so the moral of the story is RTFM. Thanks so much for your assistance and for providing such a useful tool!
